modules/Layout/Header/CategoriesDropdown/CategoryButton.tsx

```diff
@@ -21,6 +21,8 @@ function CategoryButton({ category, navigationButtonClassName }: Props) {
 
     return (
         <Button.Link
+            onResize={undefined}
+            onResizeCapture={undefined}
```

Comment:
@oleg-semyonov do you know what these are the props and why i must add them (otherwise the will be ts error)?

Reply:
I will do that, thank you for hinting the issue origin